sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/KnownMethods.cs

```diff
@@ -50,7 +50,8 @@ public static bool IsObjectEquals(this IMethodSymbol methodSymbol)
 
         public static bool IsObjectGetHashCode(this IMethodSymbol methodSymbol)
         {
-            return methodSymbol.IsOverride &&
+            return methodSymbol != null &&

```

Comment:
@michalb-sonar Yes I think so too but this refactoring doesn't belong to this PR that's why I haven't done it yet.